app/Models/Multiplayer/Room.php

```diff
@@ -37,6 +37,7 @@
  * @property int $id
  * @property int|null $max_attempts
  * @property string $name
+ * @property int|null $parent_id
```

Comment:
should be in `SeasonRoom` table instead. I also think it may be easier to manage/implement by storing the playlist letter itself, instead of linking rooms together